src/frameName.cpp

```diff
@@ -80,7 +80,8 @@ FrameName::FrameName(Arguments& args, int style, int epoch, Mutex& thread_names_
     _cache_epoch((unsigned char)epoch),
     _cache_max_age(args._mcache),
     _thread_names_lock(thread_names_lock),
-    _thread_names(thread_names)
+    _thread_names(thread_names),
+    _jni(VM::jni())
```

Comment:
There will be the same problem with native apps as above.
I suggest simply returning `NULL` from `VM::jni()` if `_vm == NULL`.